apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.spec.js

```diff
@@ -227,8 +233,9 @@ describe('notifyResidentsAboutNewsItem', () => {
                         })],
                     }),
                     meta: expect.objectContaining({
-                        title: truncate(newsItemTitle, { length: MESSAGE_TITLE_MAX_LEN, separator: ' ', omission: '...' }),
-                        body: truncate(newsItemBody, { length: MESSAGE_BODY_MAX_LEN, separator: ' ', omission: '...' }),
+                        // Check content for notifications
+                        title: expect.stringMatching(CHECK_CONTENT_MESSAGE_REGEXP),
+                        body: expect.stringMatching(CHECK_CONTENT_MESSAGE_REGEXP),
```

Comment:
I think this is a bad test. If I change `MESSAGE_TITLE_MAX_LEN` or `MESSAGE_BODY_MAX_LEN` to 1, it will pass. If I change it to 1000, it will fail

I would like the test to not let me write bad code.
It would be good if the file with tests had its own variables `MESSAGE_TITLE_MAX_LEN` and `MESSAGE_BODY_MAX_LEN` declared. This way we can be sure that nothing is broken in the sending logic